autoprof/fit/base.py

```diff
@@ -1,12 +1,13 @@
-from time import time
-from typing import Any,Union,Sequence
-import torch
 import numpy as np
-from scipy.special import gammainc
+import torch
 from scipy.optimize import minimize
+from scipy.special import gammainc
+from time import time
+from typing import Any, Union, Sequence
```

Comment:
`time` and `typing` are base python modules and so should be at the top

Reply:
Okayyy I'll be changing it ur